qemu/tests/cfg/balloon_stress.cfg

```diff
@@ -1,26 +1,30 @@
 - balloon_stress:
     type = balloon_stress
     virt_test_type = qemu
-    only Windows
     kill_vm = yes
-    repeat_times = 500
+    repeat_times = 1000
     balloon_timeout = 600
-    video_test = win_video_play
     balloon = balloon0
     balloon_dev_devid = balloon0
     balloon_dev_add_bus = yes
-    video_url = http://FILESHARE.COM/pub/section2/kvmauto/video/big_buck_bunny_480p_stereo.avi
-    x86_64:
-        program_files = "%ProgramFiles(x86)%"
-    i386:
-        program_files = "%ProgramFiles%"
-    driver_name = "balloon.sys"
-    #Disable first startup guide for windows media player
-    wmplayer_reg_cmd = "reg add HKLM\SOFTWARE\Policies\Microsoft\WindowsMediaPlayer /v GroupPrivacyAcceptance  /t REG_DWORD /f /d 00000001"
-    wmplayer_path = "${program_files}\Windows Media Player\wmplayer.exe"
-    #Install kmplayer if wmplayer is not installed default
-    kmplayer_install_cmd = "start /wait WIN_UTILS:\kmplayer\%s\KMPlayer-setup.exe /SP- /VERYSILENT"
-    kmplayer_path = "${program_files}\KMPlayer\kmplayer.exe"
-    play_video_cmd = '"%s" "%s" /play /fullscreen'
-    time_for_video = 1200
-    guest_alias = "Win2008-sp2-32:2k8\x86,Win2008-sp2-64:2k8\amd64,Win2008-r2-64:2k8\amd64,Win2012-64:2k12\amd64,Win2012-64r2:2k12\amd64"
+    Windows:
+        stress_test = win_video_play
+        video_url = http://FILESHARE.COM/pub/section2/kvmauto/video/big_buck_bunny_480p_stereo.avi
+        x86_64:
+            program_files = "%ProgramFiles(x86)%"
+        i386:
+            program_files = "%ProgramFiles%"
+        driver_name = "balloon.sys"
+        #Disable first startup guide for windows media player
+        wmplayer_reg_cmd = "reg add HKLM\SOFTWARE\Policies\Microsoft\WindowsMediaPlayer /v GroupPrivacyAcceptance  /t REG_DWORD /f /d 00000001"
+        wmplayer_path = "${program_files}\Windows Media Player\wmplayer.exe"
+        #Install kmplayer if wmplayer is not installed default
+        kmplayer_install_cmd = "start /wait WIN_UTILS:\kmplayer\%s\KMPlayer-setup.exe /SP- /VERYSILENT"
+        kmplayer_path = "${program_files}\KMPlayer\kmplayer.exe"
+        play_video_cmd = '"%s" "%s" /play /fullscreen'
+        time_for_video = 1200
+        guest_alias = "Win2008-sp2-32:2k8\x86,Win2008-sp2-64:2k8\amd64,Win2008-r2-64:2k8\amd64,Win2012-64:2k12\amd64,Win2012-64r2:2k12\amd64"
+    Linux:
+        # Use a low stress to make sure guest can response during stress
+        stress_args = "--cpu 4 --io 4 --vm 2 --vm-bytes 256M"
```

Comment:
What does this parameter do ?

Reply:
this param is used for in VMStress

Reply:
Could you please help explain how ? I didn't see the param passed to the function.
`stress_bg = utils_test.VMStress(vm, "stress")`

Reply:
In VMStress,   self.stress_args = self.params.get("stress_args", "")

Reply:
Yes, you are right. Thanks!